.github/workflows/main.yml

```diff
@@ -202,9 +202,39 @@ jobs:
       e2e-tests-logs-dump: true
     secrets: inherit
 
+  run-e2e-tests-tx-exclusion-geth-tracing:
```

Comment:
Here, I separate the transaction exclusion e2e tests from the existing e2e tests and let them run in different runners to achieve parallelism, the reason is to speed up the total time for all e2e tests given that transaction exclusion e2e tests are not related to any of the existing e2e tests, and also given that they would use a sequencer and l2 besu node with reduced trace limits (otherwise if we keep using the default trace limits, it would take quite some time or even timeout for the nodes to reject an over traces-limit transaction), so it'd be best to separate them from the existing e2e tests where the nodes are shared in each e2e test (anyway this will also be changed soon as Vic is working on a ticket to run each e2e test suite in parallel)